test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -146,6 +147,12 @@ func (s *snapshottableTestSuite) defineTests(driver TestDriver, pattern testpatt
 				framework.Failf("Error deleting claim %q. Error: %v", pvc.Name, err)
 			}
 		}()
+
+		ginkgo.By("starting a pod")
```

Comment:
@msau42 
Spoke with @boylee1111 w.r.t this piece. For PD CSI driver, it returns a storage class with delay binding and thus the need to create a pod.
As this is really to test snapshot functionality, IMHO it could be beneficial to remove this constraint.
Thought this PR is just to enable PD CSI driver e2e testing for snapshot, we could leave it here for now and think about removing it later when more test cases are added.

Reply:
I think it is fine to keep it. This test suite should work against any storageclass, with or without delayed binding.